backend/fleet_management/api.py

```diff
@@ -56,7 +56,7 @@ class DriveView(generics.ListCreateAPIView):
     required_groups = all_driver_methods
     serializer_class = DriveSerializer
     filter_backends = (filters.OrderingFilter,)
-    ordering = ('-date',)
+    ordering = ('-date', 'id',)
```

Comment:
Ordering of `-date` means that drives will be ordered by date descending, if we want to obey that rule we should use `-id` here.

Reply:
It's a misconception, it's not necessary to change this.